### ISSUE TITLE
String.Remove(0, 0) returns empty string

### ISSUE THREAD

**Issue by Suchiman** [bridgedotnet/Bridge#1447]
### Expected
`"Hello World".Remove(0, 0)` returning `"Hello World"`

### Actual
`"Hello World".Remove(0, 0)` returning `""`

### Steps To Reproduce
http://live.bridge.net/#83a008e47570da901f14cd63f4c5b7ae
```csharp
public class App
{
    [Ready]
    public static void Main()
    {
        Window.Alert("Hello World".Remove(0, 0));
    }
}
```

**Comment by geoffreymcgill** [comment]
Thanks for reporting this issue. There must be a logic error in the following js function:

https://github.com/bridgedotnet/Bridge/blob/master/Bridge/Resources/String.js#L384

**Comment by TriEdgeAI** [comment]
Shouldn't be `!count`, because that would result in `true` when `count` is 0.

**Comment by geoffreymcgill** [comment]
Looks like there are several other issues within that function. I'll try and fix and sneak into the 1.14 release.

**Comment by geoffreymcgill** [comment]
This issue has been fixed and will be included in the 1.14 release.

Also related to #1453.
